### PR TITLE
Rename all shell files to have .sh extensions.

### PR DESCRIPTION
**Pull request by ggreer**

### CODE REVIEW

File: jenkins.sh [hunk 1]
```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env bash
```
Comment (kans) [inline jenkins.sh]: was this file changed at all?

Comment (ggreer) [reply jenkins.sh]: Yeah. A lot of lines changed to add .sh to them. Sad diff algorithm. :(

Comment (ggreer) [reply jenkins.sh]: Even using git mv, it doesn't make a nice diff.

Comment (kans) [reply jenkins.sh]: sad!

**Comment by benjaminapetersen** [comment]
Not familiar enough with Jenkins to say anything meaningful about changes, but def can appreciate adding the `.sh` extension.

**Comment by ggreer** [comment]
The only changes to the jenkins.sh file was adding `.sh` to some lines, since it calls other scripts.

File: builder-run.sh [hunk 2]
```diff
@@ -31,4 +31,4 @@ docker run $ENV_STR --rm --net=host \
        $VOLUME_MOUNT \
        -v "$(pwd)":/go/src/github.com/openshift/console \
        -w /go/src/github.com/openshift/console \
-       $BUILDER_IMAGE "$@"
\ No newline at end of file
+       $BUILDER_IMAGE "$@"
```
Comment (spadgett) [inline builder-run.sh]: nit: update the usage comments at the top of this file to add the extension

**Comment by spadgett** [comment]
For reference, here's the jenkins diff:

```diff
❯ git diff master:jenkins jenkins.sh
diff --git a/jenkins b/jenkins.sh
index 7fa85f04..ab63cdbf 100755
--- a/jenkins
+++ b/jenkins.sh
@@ -85,7 +85,7 @@ builder_run () {

     mkdir -p jenkins-logs
     # shellcheck disable=SC2086
-    if ./builder-run $cmd 2>&1 | tee "jenkins-logs/$name.log"
+    if ./builder-run.sh $cmd 2>&1 | tee "jenkins-logs/$name.log"
     then
         status "$name" 'success'
         s3_upload "jenkins-logs/$name.log" "$BUILD_TAG/$name.log"
@@ -101,17 +101,17 @@ builder_run () {
 }

 set -x
-./clean
+./clean.sh

 set +e

-builder_run 'Build' 0 1 ./build
-builder_run 'Tests' 0 1 ./test
-builder_run 'GUI-Tests' 1 1 ./test-gui crud
-builder_run 'GUI-Tests-ALM' 1 0 ./test-gui alm
+builder_run 'Build' 0 1 ./build.sh
+builder_run 'Tests' 0 1 ./test.sh
+builder_run 'GUI-Tests' 1 1 ./test-gui.sh crud
+builder_run 'GUI-Tests-ALM' 1 0 ./test-gui.sh alm

 status 'Performance' 'pending'
-if DOCKER_ENV="KUBECONFIG" ./builder-run ./test-gui performance
+if DOCKER_ENV="KUBECONFIG" ./builder-run.sh ./test-gui.sh performance
 then
     description=$(cat ./frontend/gui_test_screenshots/bundle-analysis.txt)
     status 'Performance' 'success' "${description}"
@@ -128,13 +128,13 @@ GIT_SHA_MASTER=$(git rev-parse origin/master)
 IS_RELEASE_TAG=$(git describe --exact-match --abbrev=0 --tags "${GIT_SHA_HEAD}" 2> /dev/null || :)
 if [ "$GIT_SHA_HEAD" == "$GIT_SHA_MASTER" ]; then
     echo "detected master build. building & pushing images..."
-    ./push
+    ./push.sh
 elif [ ! -z "$IMAGE_TAG" ]; then
     echo "detected request to push built image using tag ${IMAGE_TAG}. building & pushing images..."
-   ./push
+   ./push.sh
 elif [ -n "$IS_RELEASE_TAG" ]; then
     echo "detected release tag ${IS_RELEASE_TAG}. building & pushing images..."
-    ./push
+    ./push.sh
 else
     echo "skipping image push. HEAD sha does not appear to be master, nor is it a release tag: $GIT_SHA_HEAD"
 fi
```

**Comment by spadgett** [comment]
Also one comment in Dockerfile-builder doesn't have the .sh extension

**Pull request merged**